research/README.MD

```diff
@@ -0,0 +1,41 @@
+# Research Directory
+Research directory will be the place to host various research work from community on Federated learning
+leveraging NVIDIA FLARE. **The code will not be maintained by NVIDIA FLARE team**, but will require Pull Request
+approval process. 
+
+## License
+By providing the code in NVFLARE repository, you will grant the research project in NVIDIA repo to be released under open source license
+Apache v2 License. 
+
+## Requirements
+Each research project should create a sub directory with following requirements
+
+* sub directory name must be in ASCII string and no longer than 35 characters long
+* Each project should include
+  * README.MD -- document must include
+    * objective 
+    * background
+    * description
+    * nvflare version used
+  * code needed to run nvflare job or Pull Request
+  * Apache2 License file
+
+## Example
+```
+sample_research$ 
+.
+├── job_configs
+    └── job1
+           ├── app_server
+                   ├── config
+                           └── config_fed_server.json
+                   └── custom
+                        └── sample_controller.py
+           └── app_client
+                   ├── config
+                           └── config_fed_client.json
+                   └── custom
+                        └── sample_executor.py
```

Comment:
So we are going to split the custom code for server and client? Currently I think they are together in one /config and /custom folder.

Reply:
Hi @ZiyueXu77 

In the "job" concept, 
The job could be

```
job
   app_for_both_server_and_client/
       config/
             config_fed_server.json
             config_fed_client.json
       custom/
             sample_controller.py
             sample_executor.py
   meta.json
```

Or

```
 job
       app_server
              config
                   config_fed_server.json
              custom
                   sample_controller.py
       app_client
               config
                   config_fed_client.json
                custom
                   sample_executor.py
```